pubsub/gossipsub/gossipsub-v1.1.md

```diff
@@ -491,6 +413,84 @@ if invalidMessageDeliveries < DecayToZero {
 
 `TBD`: We are currently developing a multiple types of simulations that will inform us on how to best recommend tunning the Scoring function. We will update this section once that work is complete
 
+### Overview of New Parameters
+
+The extensions that make up gossipsub v1.1 introduce several new application configurable
+parameters. This section lists all the new parameters along with a brief description. More detail is
+available in the discussion of the protocol extensions below.
```

Comment:
can we change "exensions below" to just "extensions"?

Reply:
done, I removed the sentence.